src/Akeneo/Pim/Structure/Bundle/EventSubscriber/Locale/ClearCacheSubscriber.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Akeneo\Pim\Structure\Bundle\EventSubscriber\Locale;
```

Comment:
```suggestion
namespace Akeneo\Pim\Structure\Bundle\EventSubscriber\Channel;
```

and probably wrongnamespace btw